bot/telegram_bot.py

```diff
@@ -370,7 +370,14 @@ async def prompt(self, update: Update, context: ContextTypes.DEFAULT_TYPE):
 
         if is_group_chat(update):
             trigger_keyword = self.config['group_trigger_keyword']
-            if prompt.lower().startswith(trigger_keyword.lower()):
+
+            raw_text = update.message.text.lower()
+            prompt_lower = prompt.lower()
+
+            if raw_text.startswith('/chat') or prompt_lower.startswith(trigger_keyword):
+                if not prompt_lower.startswith(trigger_keyword):
+                    prompt = f'{trigger_keyword} {prompt}'
+
```

Comment:
Could this work the same way?

```suggestion
            if prompt.lower().startswith(trigger_keyword.lower()) or update.message.text.lower().startswith('/chat'):
                if prompt.lower().startswith(trigger_keyword.lower()):
                    prompt = prompt[len(trigger_keyword):].strip()
```